querybuilder/query.py

```diff
@@ -1426,6 +1426,17 @@ def wrap(self, alias=None):
 
         return self
 
+    def copy(self):
+        """
+        Deeply copies everything in the query object and safely copies the connection object
+        """
+        connection = self.connection
+        del self.connection
+        copied_query = deepcopy(self)
+        copied_query.connection = connection
+        self.connection = connection
+        return copied_query
```

Comment:
It looks like the connection is actually not copied. Both the `copied_query` and `self` are using the same connection. This looks fine and since it works thats great, so could you just adjust the docstring to say that the copied query shares the original connection?